apps/myaccount/src/init/app-utils.ts

```diff
@@ -193,7 +193,8 @@ export const AppUtils: AppUtilsInterface = (function() {
                 return null;
             }
             if (_config.consoleApp && _config.consoleApp.origin) {
-                _config.consoleAppOrigin = _config.consoleApp.origin;
+                _config.consoleAppOrigin = _config.consoleApp.origin
+                    .replace(SERVER_ORIGIN_PLACEHOLDER, _config.serverOrigin);
```

Comment:
Dont we need to update this value?

```suggestion
                    .replace(SERVER_ORIGIN_URL_PLACEHOLDER, _config.serverOrigin);
```

Reply:
Yes. Accepted the suggestion.